src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentServiceTest.kt

```diff
@@ -44,6 +47,7 @@ class AppointmentServiceTest {
     communityAPIBookingService,
```

Comment:
Action Plan Session Service Test has a test as follows. Should this service not have an equivalent

  @Test
  fun `session feedback emits application events`() {}

Reply:
added